packages/loader/package.json

```diff
@@ -31,6 +34,9 @@
     "webpack",
     "loader"
   ],
+  "scripts": {
+    "test-types": "dtslint types"
```

Comment:
dtslint doesn't seem to support patterns.
I tried adding `dtslint packages/*/types` to handle all packages at once, that didn't work, so opted to add the script at the package level, and use lerna to run all of them.